platform/node/scripts/after_success.sh

```diff
@@ -4,7 +4,7 @@ set -e
 set -o pipefail
 
 if [[ -n ${PUBLISH:-} ]]; then
-    if [[ "${BUILDTYPE}" == "Release" ]]; then
+    if [[ "${BUILDTYPE}" == "RelWithDebInfo" ]]; then
```

Comment:
@springmeyer @bsudekum is this logic also feasible to go into https://github.com/mapbox/mapbox-gl-native/pull/9051 ?